Library/Formula/slib.rb

```diff
@@ -0,0 +1,24 @@
+class Slib < Formula
+  homepage "http://people.csail.mit.edu/jaffer/SLIB"
+  url "http://groups.csail.mit.edu/mac/ftpdir/scm/slib-3b4.zip"
+  version "3b4"
```

Comment:
Also: don't need to specify this as it's autodetected: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19563/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_slib___strict/